core/modules/replica/ChunksApp.h

```diff
@@ -86,8 +86,9 @@ class ChunksApp : public Application {
     /// The flag (if set) for pulling chunk disposition from Qserv workers for the combined analysis
     bool _pullQservReplicas = false;
 
-    /// The maximum timeout for the replica disposition requests
-    unsigned int _timeoutSec = 60;
+    /// The maximum timeout for the completion of requests sent to
+    /// the Replication System's and Qserv workers
+    unsigned int _timeoutSec = 0;

```

Comment:
`0` means no timeout?

Reply:
It means that a default value of the timeout would be pulled from the Configuration. I've updated the comment to make it clear.